src/dags/bedrijveninvesteringszone.py

```diff
@@ -52,7 +54,7 @@
     )
 
     # 2. Create temp directory to store files
-    mkdir = BashOperator(task_id="mkdir", bash_command=f"mkdir -p {tmp_dir}")
+    mkdir = mk_dir(Path(tmp_dir))
```

Comment:
Looks like we could get rid of some noise by having `mk_dir` work on `Union[Path, str]`...

Reply:
.... except that some paths are still constructed using string formatting. Never mind, let's tackle this in a follow-up.

Reply:
Good point. Though I have given this some thought. 

`mk_dir` is a brand new function. For anything new I'd like to use `pathlib` exclusively. It is different when I want to add `pathlib` capabilities to an existing function; then I do go from `str` to `Union[Path, str]`.

By using `pathlib` exclusively for new functions. Their call sites stand out with the explicit `Path` cast. That makes it easier to address `str` versus `Path` usage at those call sites.